### ISSUE TITLE
Open Pods and Wake Lock clarificaton

### ISSUE THREAD

**Issue by Rihcus** [adolfintel/OpenPods#122]
Hi I have been having battery issues on my Oneplus 8 running lineage os 18.1 and noticed a large number of wake requests from bluetooth.

![image](https://user-images.githubusercontent.com/30292597/125379416-7a1bb800-e35e-11eb-9985-7f17dfe835a8.png)

![image](https://user-images.githubusercontent.com/30292597/125379430-80aa2f80-e35e-11eb-9830-c31e7266d997.png)

Out of curiosity does openpods ping bluetooth or apply wake locks that prevent doze funcitionality? Also what does battery saver feature do to save battery.

As a side note the only other apps on my phone that could make that many bt requests is samsung wear and health services with my gear 3 classic I also used greenify to force doze (shallow hibernate).

 Combined with battery saver in open pods and greenify for samsung my phone battery drain dropped from 5-8% per hour to 1-2% per hour.

I guess my overall question would be do you think there is any aspect of OpenPods and its battery saver that could lead to a bluetooth wake lock and large battery drain or was it something else (ie samsung services)?

Sorry long request. 

Edit:

As a side note I use open pods with my Beats X (works except it shows up as one airpod the other airpod and case show as disconnected).

**Comment by Electric1447** [comment]
OpenPods does not have wakelock permission.
https://github.com/adolfintel/OpenPods/blob/32f8fce6a617d9944f2059ea6bacbed137b09449/OpenPods/app/src/main/AndroidManifest.xml#L5-L11

However it may still cause battery life issues with your phone, try to enable battery saver and check if it helps.

I am not exactly sure what battery saver does has it has been added before I started working on the project. I'll try to check what battery saver does and update you exactly.
